.github/workflows/slack.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     steps:
     - name: Get current star count
       run: |
-        echo ::set-env name=STARS::$(curl --silent 'https://api.github.com/repos/layer5io/layer5' -H 'Accept: application/vnd.github.preview' | jq '.watchers_count')
+        echo "STARS=$(curl --silent 'https://api.github.com/repos/layer5io/layer5' -H 'Accept: application/vnd.github.preview' | jq '.stargazers_count')" >> $GITHUB_ENV
```

Comment:
Shouldn't line: 16 be updated as we are saving the value to `GITHUB_ENV` now?

Reply:
@Nikhil-Ladha i dont think so 🤔, this seems to be working for the other repositories.

Reply:
Multiple possible right answers, I suspect. Consistency is generally a good thing for us.